docs/pages/desktop-access/rbac.mdx

```diff
@@ -76,7 +77,16 @@ host_labels:
       environment: prod
 ```
 
-For desktops discovered via LDAP, Teleport applies the following labels automatically:
+### With LDAP
+
+There are several ways that desktops discovered via LDAP can be labeled:
+
+1. Automatic `teleport.dev/` labels, applied unconditionally
```

Comment:
I think we can remove this list. I wasn't sure what "unconditionally" meant in the first item, then realized that the "Teleport applies..." sentence and the table after it would answer my questions. The same goes for the second item in the list. 

An alternative would be to add an H4-level heading for the text about unconditionally applied labels and another heading for labels based on discovery configuration so it's clearer that the list is meant as a mini-table of contents.

Reply:
"Unconditionally" was meant to indicate that you can't configure or disable this behavior, you'll get these labels no matter what.

Had we had this feature from the beginning, all of the `teleport.dev/` labels would have been unnecessary, but that's the way it goes sometimes.